dist/configs/config/server.cfg

```diff
@@ -1,21 +1,21 @@
 // Example Unvanquished dedicated server config
 
 // advertise the server on public server list
-// 0 - advertise everything
-// 1 - don't advertise but reply to status queries
-// 2 - don't reply to status queries but accept connections
-// 3 - only accept LAN connections
+// 0 - advertise everything - this will make the server viewable to everyone looking at the list
```

Comment:
Better use a comma instead of ` - ` (same for other lines).

« make the server viewable » is an ambiguous concept.

Maybe something like this is better:

> 0 - advertise everything, the server will be added to the public server list and information about it will be displayed to players